docs/framework/app-domains/how-to-install-an-assembly-into-the-gac.md

```diff
@@ -1,79 +1,55 @@
 ---
-title: "How to: Install an Assembly into the Global Assembly Cache"
-ms.date: "03/30/2017"
-helpviewer_keywords: 
+title: Install an assembly into the GAC
+ms.date: 09/20/2018
+helpviewer_keywords:
   - "assemblies [.NET Framework], global assembly cache"
   - "Gacutil.exe"
   - "strong-named assemblies, global assembly cache"
   - "global assembly cache, installing assemblies"
   - "Global Assembly Cache tool"
+  - "windows installer, global assembly cache"
 ms.assetid: a7e6f091-d02c-49ba-b736-7295cb0eb743
 author: "rpetrusha"
 ms.author: "ronpet"
 ---
-# How to: Install an Assembly into the Global Assembly Cache
-There are two ways to install a strong-named assembly into the global assembly cache (GAC):  
-  
+# How to: Install an assembly into the global assembly cache
+
+There are two ways to install a strong-named assembly into the global assembly cache (GAC).
+
+## Windows Installer
+
+[Windows Installer](/windows/desktop/Msi/installation-of-assemblies-to-the-global-assembly-cache), the Windows installation engine, is the recommended and most common way to add assemblies to the global assembly cache. Windows Installer provides reference counting of assemblies in the global assembly cache, plus other benefits. You can use the [WiX Toolset extension for Visual Studio 2017](https://marketplace.visualstudio.com/items?itemName=RobMensching.WixToolsetVisualStudio2017Extension) to create an installer package for Windows Installer.
```

Comment:
this is completely unrelated to your PR, but "recommended and most common" is a jarring phrase (at least I think so). I'd change it to just "recommended".
another unrelated nit: no comma after cache.